AdyenCard/Components/Card/CardViewController.swift

```diff
@@ -91,17 +97,20 @@ internal class CardViewController: FormViewController {
              holder: configuration.showsHolderNameField ? holderNameItem.nonEmptyValue : nil)
     }
 
+    internal var address: AddressInfo? {
+        guard configuration.showsAddressVerification != .none else { return nil }
+        return addressVerificationItem.value
+    }
+
```

Comment:
Not sure `guard` here is the best choice since it is not used as an early return and/or precondition check. It also introduces two inverse logical statements, and those become confusing quite quickly :)
Maybe like this. A bit longer, but easier to read
```
internal var address: AddressInfo? {
    if configuration.showsAddressVerification == .none {
        return nil
    } else {
        return addressVerificationItem.value
    }
}
 ```

Reply:
why not